src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java

```diff
@@ -1,121 +1,43 @@
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
 package com.qaprosoft.carina.demo;
 
-import java.util.List;
-
-import org.apache.commons.collections.CollectionUtils;
-import org.apache.commons.lang3.StringUtils;
+import com.qaprosoft.carina.core.foundation.utils.R;
 import org.testng.Assert;
 import org.testng.annotations.Test;
-import org.testng.asserts.SoftAssert;
 
 import com.qaprosoft.carina.core.foundation.IAbstractTest;
-import com.zebrunner.agent.core.annotation.TestLabel;
 import com.qaprosoft.carina.core.foundation.utils.ownership.MethodOwner;
-import com.qaprosoft.carina.core.foundation.utils.tag.Priority;
-import com.qaprosoft.carina.core.foundation.utils.tag.TestPriority;
-import com.qaprosoft.carina.demo.gui.components.FooterMenu;
-import com.qaprosoft.carina.demo.gui.components.NewsItem;
-import com.qaprosoft.carina.demo.gui.components.compare.ModelSpecs;
-import com.qaprosoft.carina.demo.gui.components.compare.ModelSpecs.SpecType;
-import com.qaprosoft.carina.demo.gui.pages.BrandModelsPage;
-import com.qaprosoft.carina.demo.gui.pages.CompareModelsPage;
-import com.qaprosoft.carina.demo.gui.pages.HomePage;
-import com.qaprosoft.carina.demo.gui.pages.ModelInfoPage;
-import com.qaprosoft.carina.demo.gui.pages.NewsPage;
+import com.qaprosoft.carina.demo.guipractice.components.NavigateMenu;
+import com.qaprosoft.carina.demo.guipractice.pages.HomePage;
+import com.qaprosoft.carina.demo.guipractice.pages.SignUpPage;
+
+import java.util.Random;
+
+public class WebSampleTest implements IAbstractTest{
+	@Test()
+	@MethodOwner(owner = "kdenysiuk")
+	public void testSignIn() {
+		//Open DemoBlaze homepage and verify
+		HomePage homePage = new HomePage(getDriver());
+		homePage.open();
+		Assert.assertTrue(homePage.isPageOpened(), "The page is not opened");
+		
+		//Open sign in label
+		NavigateMenu navigateMenu = homePage.getNavigateMenu();
+		Assert.assertTrue(navigateMenu.isUIObjectPresent(), "The navigate menu wasn't found.");
+		Assert.assertTrue(navigateMenu.isSigninButtonPresent(), "The sign in button wasn't found.");
+		SignUpPage signUpPage = navigateMenu.clickSigninButton();
+		
+		//Create an account
+		signUpPage.fillUsernameField(R.TESTDATA.get("user_profile")+ new Random().nextInt());
+		signUpPage.fillPasswordField(R.TESTDATA.get("user_password"));
+		signUpPage.clickSigninButton();
+
+		//Assert of "Welcome" label
+		Assert.assertEquals(getDriver().switchTo().alert().getText(), "Sign up successful.", "Alert is wrong");
+		getDriver().switchTo().alert().accept();
```

Comment:
where is assert of "Welcome" label? you just checked the alert

Reply:
Yes, I confused of method, the welcome label is according of the log in test, I will change the comment